src/lib/components/filter-select.svelte

```diff
@@ -2,6 +2,8 @@
   export let id: string;
   export let name: string;
   export let value: string | boolean;
+
+  let showDropdown = false;
```

Comment:
`showDropdown` is declared but its value is never read.

Reply:
will remove